docs/api.md

```diff
@@ -45,450 +45,528 @@
+-   `cb` **[function][79]** (err, res) style callback function
 
 ## Import
 
-[lib/import.js:45-238][103]
+[lib/import.js:45-238][108]
 
--   **See: [Hecate Documentation][104]**
+-   **See: [Hecate Documentation][109]**
 
 ## validateGeojson
 
-[util/validateGeojson.js:23-56][105]
+[util/validateGeojson.js:23-57][110]
 
 Ensure geometries are valid before import
 
 ### Parameters
 
--   `opts` **[Object][73]** Options object (optional, default `{}`)
-    -   `opts.ignoreRHR` **[boolean][78]** =false Ignore Right Hand Rule errors
-    -   `opts.schema` **[Object][73]** JSON Schema to validate properties against
-    -   `opts.ids` **[boolean][78]** If false, disable duplicate ID checking
+-   `opts` **[Object][78]** Options object (optional, default `{}`)
+    -   `opts.ignoreRHR` **[boolean][83]** =false Ignore Right Hand Rule errors
+    -   `opts.schema` **[Object][78]** JSON Schema to validate properties against
+    -   `opts.ids` **[boolean][83]** If false, disable duplicate ID checking
 
 ## validateFeature
 
-[util/validateGeojson.js:70-159][106]
+[util/validateGeojson.js:71-213][111]
 
 Validate a single feature
 
 ### Parameters
 
--   `line` **([Object][73] \| [String][80])** Feature to validate
--   `options` **[Object][73]** 
-    -   `options.ignoreRHR` **[boolean][78]** Ignore winding order
-    -   `options.schema` **[Function][74]** AJV Function to validate feature properties against a JSON Schema
-    -   `options.linenumber` **[number][107]** Linenumber to output in error object
-    -   `options.ids` **[Set][108]** Set to keep track of feature id duplicates
+-   `line` **([Object][78] \| [String][85])** Feature to validate
+-   `options` **[Object][78]** 
+    -   `options.ignoreRHR` **[boolean][83]** Ignore winding order
+    -   `options.schema` **[Function][79]** AJV Function to validate feature properties against a JSON Schema
+    -   `options.linenumber` **[number][112]** Linenumber to output in error object
+    -   `options.ids` **[Set][113]** Set to keep track of feature id duplicates
 
-Returns **[Array][79]** Array of errors (empty array if none)
+Returns **[Array][84]** Array of errors (empty array if none)
 
 ## Revert
 
-[lib/revert.js:14-243][109]
+[lib/revert.js:10-127][114]
+
+### deltas
+
+[lib/revert.js:38-126][115]
+
+Revert a given set of deltas
+
+#### Parameters
+
+-   `options` **![Object][78]** options for making reversion of a set of deltas (optional, default `{}`)
+    -   `options.start` **[Number][112]** Incusive start delta ID to revert
+    -   `options.end` **[Number][112]** inclusive end delta ID to revert
+    -   `options.output` **[Stream][86]?** Stream to write line-delimited GeoJSON to
+-   `cb` **[function][79]** (err, res) style callback function
+
+## inverse
+
+[util/revert.js:48-109][116]
 
-## getDelta
+Given the feature history for a single feature,
+revert the last feature in the history to that
+of the second last
```

Comment:
It took me a bit to wrap my head around this, even knowing how the system works. Maybe use the word recent instead of last? When I read 'last' my mind went to the oldest feature in the list.